src/styles/_ProductDetailOrderForm.scss

```diff
@@ -98,17 +101,9 @@
       top: -10px;
       right: -10px;
     }
-  }
-}
 
-@media (min-width: 800px) {
-  .details-order-form__selection-pair {
-    select:hover {
-      cursor: pointer;
-      border: 2px solid v.$primary-accent-color;
+    .details-order-form__hidden {
+      display: none;
     }
   }
-  .details-order-form__submit-btn:hover {
```

Comment:
The hover should stay in the wide screen media query.

Reply:
I respectfully disagree. If I have the window open on my computer screen but I have it minimized to only take up a fraction of the screen, the hover states should still work. Having hovers does not hurt anything in mobile view.

Reply:
Definitely need a tie breaker on this. 800 px is REALLY narrow. If a user misses hover states on that narrow of a screen, I think it's ok. It's not even your laptop screen width.

Whereas, having hover states on all screens would definitely contribute to a poor experience for the mobile user who'd see hover state, and then active state. 

With the hover state active on mobile screen, it looks like this:
![Screenshot 2023-09-03 at 7 32 46 PM](https://github.com/EverUse/EverUse-FE/assets/122247155/831498bb-0653-44d9-90e9-12b836f1ff20)
when it should look like this
![Screenshot 2023-09-03 at 7 32 46 PM](https://github.com/EverUse/EverUse-FE/assets/122247155/a5bfa16e-f53b-4012-9e09-604b5ff86f16)

Reply:
I don't know what I'm looking at tbh, is it the color should be darker but it is always stuck in the hover color?